notebooks/ltx-video/ltx-video.ipynb

```diff
@@ -0,0 +1,678 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    pipe.text_encoder = TextEncoderWrapper(compiled_text_encoder, text_encoder_dtype)
</pre><p>could you please check that original pytorch models were unloaded from memory when you override them? I think it may lead to extra RAM during pipeline execution if they still exist in memory</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2686/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>